lib/components/src/Zoom/ZoomElement.tsx

```diff
@@ -20,16 +21,52 @@ type ZoomProps = {
   children: ReactElement | ReactElement[];
 };
 
+const useMutationObserver = ({
```

Comment:
Can this live in this file or should this be moved somewhere else?

Reply:
If this is the only place we use a MutationObserver, I'm fine with it being in this file.